crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -263,9 +263,6 @@ async fn two_limit_orders_test(web3: Web3) {
         buy_amount: to_wei(2),
         valid_to: model::time::now_in_epoch_seconds() + 300,
         kind: OrderKind::Sell,
-        app_data: OrderCreationAppData::Full {
```

Comment:
This was here before in order to test the correctness of the `appData`, since we are having a specific test for it, I am removing it from here to avoid confusions.